conans/assets/templates/new_v2_bazel.py

```diff
@@ -23,13 +24,18 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def validate(self):
+        if self.settings.os in ("Windows", "Macos") and self.options.shared:
+            raise ConanException("Windows and Macos needs extra BUILD configuration to be able "
+                                 "to create a shared library. Please, check this reference to "
+                                 "know more about it: https://bazel.build/reference/be/c-cpp")
+
     def layout(self):
         bazel_layout(self)
 
     def build(self):
         bazel = Bazel(self)
-        bazel.configure()
-        bazel.build(label="//main:{name}")
+        bazel.build()
```

Comment:
Would it be possible in the template to create different targets and do something like:

```
        bazel = Bazel(self)
        target_suffix = "_shared" if self.options.shared else ""
        bazel.build(target=f"//main:mypackage{target_suffix}")
```

to enable creating both shared an static libraries?